diagnostic.md

```diff
@@ -49,7 +57,9 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
+@function tint($cornflower, 20%) {
+  @return mix(white, $cornflower, $percentage);
+}
```

Comment:
That's how you would set up the function, but to call it we do `tint($cornflower, 20%);`